projects/indicators/src/modules/chevron/chevron.component.ts

```diff
@@ -6,15 +6,29 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
   templateUrl: './chevron.component.html',
 })
 export class SkyChevronComponent {
-  @Output()
-  public directionChange = new EventEmitter<string>();
+  @Input()
+  public ariaControls: string;
+
+  @Input()
+  public ariaLabel: string;
 
   @Input()
   public direction = 'up';
 
   @Input()
   public disabled = false;
 
+  @Output()
+  public directionChange = new EventEmitter<string>();
+
+  public ariaExpanded(): boolean | undefined {
```

Comment:
Using a method in the template can cause performance issues; what do you think about making this a public property that gets set whenever `direction` is set?

Side note: Should we disable the `direction` input if the `disabled` value is set to `true`? 

We may need a setter on the `direction` input that updates the `ariaExpanded` property, and takes into account the `disabled` value (if you feel that's necessary).

Reply:
Agree on making a public property instead of a method. Should have thought of that.

I don't think disabled needs to be coupled to the direction and/or expanded, because you can still have an expanded thing (think tiles) that is disabled. Same goes for a collapsed thing.